lib/index.js

```diff
@@ -0,0 +1,48 @@
+var GitHubApi = require("github");
+var stream = require('stream');
+
+var reposForOrg = function(options, done) {
+  var github = new GitHubApi({
+    debug: false,
+    protocol: "https",
+    host: "api.github.com", 
+    pathPrefix: null,
+    timeout: 5000,
+    followRedirects: false
+        
+  });
+
+  github.authenticate({
+    type: "oauth",
+    token: options.token
+  })
+
+  var readStr = stream.Readable({ objectMode: true });
+    
+  github.getAllPages(github.repos.getForOrg, {
+    org: options.org
+  }, function(error, response) {
+    if (error) {
+      return done(error)
+    }
+    var forked = false;
+    if (options.forked) {
+      forked = true;
+    }
+    readStr._read = function (){
+      for (var i = 0; i < response.length; ++i) {
+        if (!forked && response[i].fork){
+          continue;
+        }
+        var obj = {};
+        obj['name'] = response[i].name;
+        obj['url'] = response[i].html_url;
+        readStr.push(obj);
```

Comment:
For each iteration of this loop, you update the same object instance and push it into the stream without making a copy. Thus consumers of the stream will have to make a copy if they wish to retain a reference to the object. (You are producing a stream of the same object repeatedly)

Let's double-check the recommended semantics for an objectMode stream to see if this is right or wrong behavior.